File: pythonFiles/bubbleSort.py
#This code implements bubble sort in Python
list=[3,4,5,2]
print(list)
swapped=True

while(swapped==True):
	swapped=False
	for i in range(0,len(list)-1):
		if list[i+1]<list[i]:
			temp=list[i]
			list[i]=list[i+1]
			list[i+1]=temp
			swapped=True

print(list)

File: c_programming/bubbleSort.c
//Program to implement bubble Sort
#include<stdio.h>
int main()
{
    int a[5]={4,2,1,5,6};
    int swapped=1,temp;
    
    while(swapped)
    { 
        swapped=0;
        for(int i=0;i<4;i++)
        {
            if(a[i]>a[i+1])
            {
            temp=a[i];
            a[i]=a[i+1];
            a[i+1]=temp;
            swapped=1;
            }
        }
    }
    for(int j=0;j<5;j++)
    {
        printf("\n%d",a[j]);
    }
}
